tests/fixtures/api_controller.py

```diff
@@ -102,7 +102,7 @@ def set_base_url(self):
         base_url.value = "http://test-circulation-manager/"
 
     def circulation_manager_setup_with_session(
-        self, session: Session, overrides: ControllerFixtureSetupOverrides = None
+        self, session: Session, overrides: ControllerFixtureSetupOverrides | None = None
```

Comment:
`ControllerFixtureSetupOverrides | None` <-> `Optional[ControllerFixtureSetupOverrides]`?

Reply:
Yes, those two are equivalent. This new `|` syntax is enabled by importing `from __future__ import annotations` at the top of the file.

In any file that does this future import the pyupgrade linter prefers the `|` union notion over the old `Optional[]` form, see [docs here](https://github.com/asottile/pyupgrade#pep-604-typing-rewrites).